deploy/operator.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubedirector
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: kubedirector
+  template:
+    metadata:
+      labels:
+        name: kubedirector
+    spec:
+      serviceAccountName: kubedirector
+      containers:
+        - name: kubedirector
+          # Replace this with the built image name
+          image: REPLACE_IMAGE
+          command:
+          - kubedirector
```

Comment:
Change to use the following
```          command:
          - "/bin/sh"
          args:
          - "-c"
          - "mkfifo /tmp/fifo; (/root/kubedirector &> /tmp/fifo) & while true; do cat /tmp/fifo; done"
```

Reply:
The file already looks like that so I'm not sure what you want me to change?

Reply:
I think Swami made this comment before your most recent changes.